streampipes-model/src/main/java/org/apache/streampipes/model/util/ElementIdGenerator.java

```diff
@@ -21,7 +21,7 @@
 
 public class ElementIdGenerator {
 
-  private static final String prefix = "urn:streampipes.apache.org:spi:";
```

Comment:
Could it be that we need an update script, or does this change only affect newly created IDs?

Reply:
Thanks for reviewing! This should not affect existing IDs.
I will create a migration script to cleanup existing models in the database as well.